src/main/java/org/osc/manager/ism/entities/SecurityGroupInterfaceEntity.java

```diff
@@ -38,106 +38,108 @@
 @Entity
 @Table(name = "SECURITY_GROUP_INTERFACE")
 public class SecurityGroupInterfaceEntity implements ManagerSecurityGroupInterfaceElement {
-
-	@Id
+    @Id
     @GeneratedValue
     @Column(name = "id")
-	private Long id;
+    private Long id;
 
-	@Column(name = "name", unique = true, nullable = false)
+    @Column(name = "name", unique = true, nullable = false)
     private String name;
 
-	@ManyToMany(fetch = FetchType.EAGER)
-	@JoinTable(name = "SECURITY_GROUP_INTERFACE_POLICY",
-			joinColumns = @JoinColumn(name = "sgi_fk", referencedColumnName = "id"),
-			inverseJoinColumns = @JoinColumn(name = "policy_fk", referencedColumnName = "id"))
-	private Set<PolicyEntity> policies = new HashSet<>();
+    @ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(name = "SECURITY_GROUP_INTERFACE_POLICY",
+            joinColumns = @JoinColumn(name = "sgi_fk", referencedColumnName = "id"),
+            inverseJoinColumns = @JoinColumn(name = "policy_fk", referencedColumnName = "id"))
+    private Set<PolicyEntity> policies = new HashSet<>();
 
-	@Column(name = "tag", nullable = true)
+    @Column(name = "tag", nullable = true)
     private String tag;
 
-	@OneToOne(fetch = FetchType.EAGER)
-	@JoinColumn(name = "security_group_fk", nullable = true, foreignKey = @ForeignKey(name = "FK_SGI_SG"))
-	private SecurityGroupEntity securityGroup;
+    @OneToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "security_group_fk", nullable = true, foreignKey = @ForeignKey(name = "FK_SECURITY_GROUP"))
```

Comment:
revert the FK name to FK_SGI_SG. FK_SECURITY_GROUP does not exist. This will fail startup of the plugin.

Please test the changes with new database. I've seen the issue where once a DB is created all tests are run against that db even thoguh you have updated the schema.